quadtree/src/lib.rs

```diff
@@ -246,4 +332,13 @@ mod tests {
             assert_eq!(&NodeId::from_str(id).unwrap().to_string(), id);
         }
     }
+
+    #[test]
+    fn test_conversion() {
+        for id in &["r", "r0", "r123323"] {
+            let node_id = NodeId::from_str(id).unwrap();
+            let spatial_node_id = SpatialNodeId::from(node_id);
+            assert_eq!(NodeId::from(spatial_node_id), node_id);
+        }
+    }
```

Comment:
And maybe let's add another unit test for your `SpatialNodeId::neighbor` function?

Reply:
Puh, were should I start and end here? I think the function is trivial enough not to test it...

Reply:
yeah you are probably right :P